SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -388,13 +388,21 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	 */
 	UPROPERTY(EditAnywhere, Config, Category = "Logging", AdvancedDisplay)
 	float ActorMigrationLogRate;
+
 	/*
 	 * -- EXPERIMENTAL --
 	 * This will enable event tracing for the Unreal client/worker.
 	 */
 	UPROPERTY(EditAnywhere, Config, Category = "Event Tracing")
 	bool bEventTracingEnabled;
 
+	UPROPERTY(EditAnywhere, Config, Category = "Event Tracing",
+			  meta = (EditCondition = "bEventTracingEnabled", ClampMin = 0.0f, ClampMax = 1.0f))
+	float SamplingProbability;
```

Comment:
Worth adding a comment description for this describing what it does, and for the one below.